utils/create-component.js

```diff
@@ -0,0 +1,56 @@
+require("colors");
```

Comment:
This feels like it might belong in the `scripts/` folder? That's where we have our other helper scripts.